lib/stream-pyramid.js

```diff
@@ -65,6 +65,12 @@ Pyramid.prototype._params = function(callback) {
 
         stream.bboxes = {};
         var bounds = limitBounds(stream.bounds);
+
+        // Validate bounds values
+        for (var b = 0; b < bounds.length; b++) {
+            if (isNaN(bounds[b])) return stream.emit('error', new Error('Invalid extent value'));
```

Comment:
`isNaN(null) === false`. You may need to get a little trickier in your condition that the value is numeric. This isn't a one-liner in javascript unfortunately...